packages/fast-components/src/slider/slider.styles.ts

```diff
@@ -0,0 +1,129 @@
+import { css } from "@microsoft/fast-element";
+import { disabledCursor, display } from "../styles";
+import { focusVisible } from "../styles/focus";
+import { SystemColors } from "../styles/system-colors";
+import { heightNumber } from "../styles/size";
+
+export const SliderStyles = css`
+    :host([hidden]) {
+        display: none;
+    }
+
+    ${display("inline-grid")} :host {
+        --thumb-size: calc(${heightNumber} * 0.5);
+        --thumb-translate: calc(var(--thumb-size) * 0.5);
+        --track-overhang: calc((var(--design-unit) / 2) * -1);
+        align-items: center;
+        outline: none;
+        width: 100%;
+        margin: calc(var(--design-unit) * 1px) 0;
+        user-select: none;
+    }
+    .slider {
+        box-sizing: border-box;
+        border-radius: calc(var(--corner-radius) * 1px);
+        outline: none;
+        cursor: pointer;
+    }
+    .foreground-track {
+        display: none;
```

Comment:
Is this mean't to always be hidden?

Reply:
this is for the other modes that aren't being implemented in this first version so I should maybe just get rid of it

Reply:
removed it and other stuff related to dual mode which won't be in this pr